Dockerfile

```diff
@@ -38,7 +38,7 @@ FROM ${RUNNER_IMAGE} as runner
 
 ARG EXTRA_RUNNER_APK_PACKAGES
 
-RUN apk add --no-cache --update ncurses dbus gmp libsodium gcc \
+RUN apk add --no-cache --update ncurses dbus gmp libsodium gcc tpm2-tss-esys tpm2-tss-fapi tpm2-tss-mu tpm2-tss-rc tpm2-tss-tcti-device\
```

Comment:
All of these extra packages are supposed to go into the `EXTRA_RUNNER_APK_PACKAGES` build-arg